backend/typescript/utilities/errorMessageUtil.ts

```diff
@@ -0,0 +1,7 @@
+const getErrorMessage = (error: unknown): string => {
+  return error instanceof Error
+    ? getErrorMessage(error)
```

Comment:
doesn't this cause an infinite loop :o 
```suggestion
    ? error.message
```

Reply:
lol yes thats why you dont do replace all